spec/Section 5 -- Validation.md

```diff
@@ -446,7 +446,7 @@ SameResponseShape(fieldA, fieldB):
 - If {typeA} or {typeB} is Scalar or Enum:
   - If {typeA} and {typeB} are the same type return true, otherwise return
     false.
-- Assert: {typeA} and {typeB} are both composite types.
+- Assert: {typeA} and {typeB} are both Object, Interface or Union.
```

Comment:
This could be considered as `(a is object and b is object) or (a is interface and b is interface) or (a is union and b is union)`. I think it's clearer without this change; alternatively use "each":

```suggestion
- Assert: each of {typeA} and {typeB} are an Object, Interface or Union.
```